fix(core): whitelabel login page

BPMJoannette opened the pull request:
Login custom page title, logo and css file were loaded too late to be applied correctly.

BPMJoannette committed:
fix(core): whilelabel login page

JijeshP commented:
@BPMJoannette Also when you refresh studio / admin still you can see default page title for few seconds.

allardy commented:
@JijeshP  Yeah, I think we could just remove the <title> tag from index.html in the admin panel to fix that issue. Can you try that @BPMJoannette  ? 

Since the HTML file has a default title and loads the env file which will override the title, there's a small delay until everything is set up. I think having it empty on startup would make sense

BPMJoannette committed:
fix(core): whitelabel login page

Removing the default title in the index waits for Botpress to pass the title, and Botpress uses the branding configuration, else "Botpress Admin Panel" at that point, so that resolves @JijeshP 's issue.

BPMJoannette commented:
@allardy @JijeshP 
Made the test, removing the title line from index.html resolved this

slvnperron commented:
LGTM
